adding phluesky to projects (#230)
bluesky-social/atproto-website: content/community/projects.md

#### PHP

- [socialweb/atproto-lexicon](https://github.com/socialweb-php/atproto-lexicon): Parses and resolves Lexicon schemas; useful for code generation

> - [potibm/phluesky](https://github.com/potibm/phluesky): An small PHP library for posting to Bluesky, new, not stable 

Reviewer: ```suggestion
- [potibm/phluesky](https://github.com/potibm/phluesky): a small library for posting to Bluesky, new, not stable 
```
